### PR TITLE
[NUI] Window operations can get detail result

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

**Comment by rabbitfor** [comment]
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/258201/
It works with this patch. Please do not merge before the dali patch merged.

**Comment by dongsug-song** [comment]
@rabbitfor 
Hello,
I put the **\*\*NO MERGE\*\*** label on this PR now.
You can remove **\*\*NO MERGE\*\*** label for yourself when this PR is ready to be reviewed and merged.
(**\*\*NO MERGE\*\*** label has no restriction to submit PR or to edit this patch. Just for notification and marking)
Thank you.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**
